src/bsp/src/stm32/hal/decawave/platform/stm32f429zi/include/deca_port.h

```diff
@@ -34,6 +40,24 @@ void deca_setFastRate();
  */
 void deca_init();
 
+/**
+ * @brief Choose which decawave is selected for the next SPI communication
+ * @param selectedDevice The device to select
+ */
+void deca_selectDevice(decaDevice_t selectedDevice);
```

Comment:
Wouldn't it be easier to have two getter for a static struct variable. One getter for the decaA and another to the decaB. Then pass that struct around. The struct would contain the necessary info, so the NssPort and Pin in this case. Avoiding to call Select device each time and just get the struct directly?

Reply:
Good idea, changed it for a structure pointer that is passed around